client/src/components/LobbySettings.svelte

```diff
@@ -0,0 +1,141 @@
+
+  function timeToMinutes(time: number) {
+    let minutes = Math.floor(time / 60);
+    let seconds = time - minutes * 60;
+
+    return ": " + minutes + ":" + seconds;
+  }
+
+  async function apply(catfish: number, prompt: number, chat: number, vote: number) {
+    try {
+      await applyLobbySettings({
+        code: lobbyCode,
+        lobbySettings: {
+          catfishAmount: catfish,
+          promptTime: prompt,
+          chatTime: chat,
+          voteTime: vote,
+        },
+      });
+    } catch (err) {
+      errorMessage = err instanceof Error ? err.message : String(err);
+    }
+  }
+</script>
+
+<main>
+  <Dialog
+    bind:open={showLobbySettings}
+    fullscreen
+    aria-labelledby="rules-dialog-title"
+    aria-describedby="rules-dialog-content"
+    ><Header>
+      <Title>Lobby Settings</Title>
+      <IconButton action="close" class="material-icons">close</IconButton>
+    </Header>
+    <Content>
+      <div class="settings">
+        <FormField align="end" style={"width:100%"}>
+          <Slider
+            bind:value={catfishValue}
+            min={1}
+            max={3}
+            step={1}
+            input$aria-label="Discrete slider"
+            style={"width:80%;"}
+          />
+          <span slot="label">Catfish Amount: {catfishValue}</span>
+        </FormField>
+
+        <FormField align="end" style={"width:100%"}>
+          <Slider
+            bind:value={promptTimerValue}
+            min={minimumTimes.get("PROMPT")}
+            max={maximumTimes.get("PROMPT")}
+            step={5}
+            input$aria-label="Discrete slider"
+            style={"width:80%;"}
+          />
+          <span slot="label">{sliderLabels.get("PROMPT")} {timeToMinutes(promptTimerValue)}</span>
+        </FormField>
+
+        <FormField align="end" style={"width:100%"}>
+          <Slider
+            bind:value={chatTimerValue}
+            min={minimumTimes.get("CHAT")}
+            max={maximumTimes.get("CHAT")}
+            step={5}
+            input$aria-label="Discrete slider"
+            style={"width:80%;"}
+          />
+          <span slot="label">{sliderLabels.get("CHAT")} {timeToMinutes(chatTimerValue)}</span>
+        </FormField>
+
+        <FormField align="end" style={"width:100%"}>
+          <Slider
+            bind:value={voteTimerValue}
+            min={minimumTimes.get("VOTE")}
+            max={maximumTimes.get("VOTE")}
+            step={5}
+            input$aria-label="Discrete slider"
+            style={"width:80%;"}
+          />
+          <span slot="label">{sliderLabels.get("VOTE")} {timeToMinutes(voteTimerValue)}</span>
+        </FormField>
+      </div>
+    </Content>
+    <Actions class="settings">
+      <Button on:click={() => apply(catfishValue, promptTimerValue, chatTimerValue, voteTimerValue)} action="close"
+        ><Label>Apply Settings</Label></Button
+      >
+    </Actions>
+  </Dialog>
+  <Button
+    on:click={() => {
+      showLobbySettings = true;
+    }}
```

Comment:
if you do `on:click={() => (showLobbySettings = true)}` then you can put it on one line